dascore/viz/wiggle.py

```
@@ -50,12 +52,51 @@ def _format_y_axis_ticks(ax, offsets, other_axis_ticks, max_ticks=10):
+    # values for axis which is connected
+    connect_axis_ticks = patch.coords.get_array(dim)
+    # values for y axis (not connected)
+    other_axis_ticks = patch.coords.get_array(other_dim)
+    offsets, data_scaled = _get_offsets_factor(patch, dim, scale, other_axis_ticks)
+    # now plot, add labels, etc.
+    ax.plot(connect_axis_ticks, data_scaled, color=color, alpha=alpha)
+    # shade the part of each wiggle above its offset if desired
+    if shade:
+        _shade(offsets, ax, data_scaled, color, connect_axis_ticks)
+    _format_y_axis_ticks(ax, offsets, other_axis_ticks)
+    for dim, x in zip(patch.dims, ["x", "y"]):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)